GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1724,6 +1730,20 @@ public void ShowFilteredBranches()
             PerformRefreshRevisions();
         }
 
+        public void ShowCurrentBranchOnly()
```

Comment:
[nit] Can we please retain `ShowCurrentBranchOnly` where it was (i.e., lines:1690-...) and place `ShowReflogs` method here (i.e., after `ShowFilteredBranches`)?

Reply:
I reordered them to match how they are used, but can revert.